Tests/Receipt/Responses/PrepaidCardReceiptResponse.json

```diff
@@ -0,0 +1,333 @@
+{
+    "count": 20,
```

Comment:
there's no count, offset, limit in prepaidCardReceipt response

Reply:
Yes, removed